blacklist/build.gradle

```diff
@@ -29,13 +29,7 @@ buildscript {
     }
 }
 
-repositories {
-    mavenLocal()
-    jcenter()
-    mavenCentral()
-    maven { url 'https://jitpack.io' }
-    maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda' }
-}
+apply from: '../repositories.gradle'
```

Comment:
Whilst replacing all the repositories blocks makes sense from a maintanance point of view, from the point of view of someone checking out the samples repo and just wanting to take one of the examples out into a seperate repo to build on it (i tried to do it the other night with the obligation sample), it just makes things more complicated, so I'd rather keep these in for now if possible. Understand it's also a case with the version numbers, although it's probably more obvious how to deal with those.

Reply:
But all the samples are in the _same_ GitHub repo, so you can't just check out one of them...?

Reply:
I mean, I check out the whole samples repo, and I want to use the obligation app as the basis of a project, so I copy that folder (only) into a new project and then I start getting weird 'can't find this/can't find that' gradle errors. Especially since the example cordapp is in there which is where we direct everyone to start with. That's why the samples repo doesn't have just one root build gradle.

Reply:
Do file-systems on Windows support symbolic links out of the box? It appears that [GitHub does support them](https://stackoverflow.com/questions/954560/how-does-git-handle-symbolic-links), and having each sample project symlink to `repositories.gradle` in the root project sounds like a reasonable compromise.

Reply:
Nope it doesn't work sorry, well not out of the box. I'm sure I could get it working with playing around with administrator settings and the like, but from the point of view of a new Corda user I think it's too much faffing and will put people off.

Reply:
I've removed the symlinks but kept all `repositories.gradle` files because they're _still_ easier to manage if they're broken out of `build.gradle`.